src/components/Profile/Profile.js

```diff
@@ -15,11 +15,11 @@ const Profile = (props) => {
 };
 
 Profile.defaultProps = {
-  profile: PropTypes.number,
+  profile: PropTypes.object,
```

Comment:
"object" is to vague, consider using "PropTypes.shape()".

Reply:
I would prefer to do this upon completion of priority tasks